ios/RNSScreenStackHeaderConfig.mm

```diff
@@ -658,13 +580,19 @@ + (void)updateViewController:(UIViewController *)vc
           }
 #endif
         }
+
+#endif // RN_FABRIC_ENABLED
+        break;
       }
       case RNSScreenStackHeaderSubviewTypeBackButton: {
+#ifdef RN_FABRIC_ENABLED
+        RCTLogWarn(@"Back button subivew is not yet Fabric compatible in react-native-screens");
+#endif
         break;
+        ;
```

Comment:
Why is this semicolon here? :D

Reply:
[b0f1141](https://github.com/software-mansion/react-native-screens/pull/1418/commits/b0f1141666ca2b7ef55d5f660a3f855bef7970d5)